src/release.js

```diff
@@ -61,7 +51,9 @@ module.exports = async function ({ github, context, inputs }) {
   }
 
   const run = runSpawn()
-  const branchName = `release/${version}`
+  const branchName = monorepoPackage
+    ? `release/${monorepoPackage}-${version}`
```

Comment:
could you `require('./package.json').name` here to get the specific package name?

Reply:
I'm not sure this would be a good idea since the package name could include a namespace such as @babel/preset-env